test/modules/TwinTester/Settings.cs

```diff
@@ -29,8 +29,8 @@ static Settings Create()
                 configuration.GetValue<string>("TargetModuleId", moduleId),
                 configuration.GetValue<int>("TwinUpdateSize", 1),
                 configuration.GetValue<TimeSpan>("TwinUpdateFrequency", TimeSpan.FromSeconds(10)),
-                configuration.GetValue<TimeSpan>("TwinUpdateFailureThreshold", TimeSpan.FromMinutes(1)),
-                configuration.GetValue<TimeSpan>("EdgeHubRestartFailureTolerance", TimeSpan.FromMinutes(1)),
+                configuration.GetValue<TimeSpan>("TwinUpdateFailureThreshold", TimeSpan.FromMinutes(2)),
+                configuration.GetValue<TimeSpan>("EdgeHubRestartFailureTolerance", TimeSpan.FromMinutes(2)),
```

Comment:
We want to give twin updates two minutes to come in, to avoid transient network issues failing the tests.

Reply:
EdgeHub restart tolerance needs to also increase. This is because when edgehub restarts it will start closing upstream subscriptions on behalf of test modules. Therefore, we want to give any update a full two minutes to come through without the edgehub restart process interfering.

Reply:
Do we have a stat on this?

Reply:
What do you mean by a stat?